test/fixtures/logging-job.yaml.tmpl

```diff
@@ -9,5 +9,5 @@ spec:
       restartPolicy: Never
       containers:
       - name: smoketest-logging
-        image: nginxinc/nginx-unprivileged:1.21-alpine
-        command: ["curl", "-s", "-XGET", "{{ .searchTerm }}", "-H", 'Content-Type: application/json', "-d", '{ "query": { "bool": { "must": [ { "match": { "kubernetes.namespace_name.keyword": "{{ .namespace }}" } } ] } } }']
+        image: ministryofjustice/curl-jq:1
```

Comment:
Is this image already maintained by moj or something new CP added? if thats something we manage, would be good to add a comment on the dockerfile source so we know how to build it

Reply:
good point I've added the dockerfile to the test dir, so we have a copy of the source and have updated the readme with reference to it.